cowprotocol/services
# Publish auction on S3 rather than printing it

# Description
This PR gets rid of the 5MB auction printing log and instead uploads the auction in the background to an S3 bucket.

# Changes
- [x] Only print auction id (no longer full auction)
- [x] Upload auction to S3 in background

## How to test
Change an e2e test to contain our testing bucket config. Run test, see file [being uploaded](https://staging-bucket-133876976458.s3.eu-central-1.amazonaws.com/felix_local/6.json?response-content-disposition=inline&X-Amz-Security-Token=IQoJb3JpZ2luX2VjEJD%2F%2F%2F%2F%2F%2F%2F%2F%2F%2FwEaCWV1LXdlc3QtMiJHMEUCIAwTaLf%2BriJI%2Fllsp%2F7vjdw%2FEh1gI5Q1TJmCbmfb0j8gAiEA5OqnKjczyXHYNy86aI1DHyo27heVigFysMEg7rHKXdEqzwMIGRABGgwxMzM4NzY5NzY0NTgiDCZf7AxG4oIKUv922CqsAyKbMyquqRBErm2QX3hhGlXMXeIHO7Wjt46aoyMJuEUZUBlLg2E1oT7R7lwsUuuyDZQfb1JbmtpJRjRYaCjMSOGfBbyTtVoxluzBwM2S1AuZTEdPEvBBXIE8B8KhQbESRB0SxJyyB58r0Gw9PxcMo8oKqa%2F%2F5zLJTDFkb94zqmtvscYkM2veAoriuuxr8k6DP2NB7w09mGRcHx%2BWmIByarB3D9VXMTqNIjw44tOOUa1AWNjClVQJZoNvWKUsLp2AwvRjmanpdQrbo5QO355X%2BTqH%2F2FNUm7BiFxGyvcnaruquxffv2xCx7j2%2FpcyQA44o0AEQiiRdPmJpYLCp6c9QcyxA71eJ8Y5W2WvCpN6JNoDJ4qpuEFNzOpAxF%2FVt01xwXpdI8JlUe5boSCPBuVCbmDDgtDYdGi%2FRP07UzpnZD8LDYKS4orWybmMC3QhcSIlSMGa%2F0U4%2FdjNxCA0K%2FsEXzlVJ167SW8QTazE%2FavcngiaINPI8MA9MWjFZmQW3I2w3%2BCYDyG8AugalKyj%2FdTEKTj6BXOnXyg7Ck178%2Fre42aVSyH%2FTiaExQg%2BKERkMJrFkawGOpQC9Vwdg8FfG2DvDdEdBVklqMtO9sWn%2FkIpcqMt8UnijqrMa3e4K%2FhImEA9L6QFsNRzfI%2BfX9OEuUru2ZW%2FdvX19hY59jznTyKgcTWEszdGy39ymiIqxsCPu%2FccxsbaKvxiwM2zSFIfR0LdSJ2u62lmkJJ2xLW5YG75%2FoHpVzunyeQKazsRcoxbbzLWDXOCch6ZF5plT0uwjmlTsGiwmBG9YwEnmz%2BnDjdZ68GO6hD%2Fh8pPghL6LMYjLbCKFPw%2FQjfMDNZMDtlZ7D%2Bw%2B1VsZoe3JMikI0nfqvD%2FsAiuRFBwKHtWdQeyM7hWsRYuO%2FhaBUWk2aKGUszXsn8z2k1dP6JiPdFpmYxx6vzrU%2Fv%2BnFaATQhRFtLf&X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Date=20231221T175408Z&X-Amz-SignedHeaders=host&X-Amz-Expires=300&X-Amz-Credential=ASIAR6K5QRNFCB3CNA27%2F20231221%2Feu-central-1%2Fs3%2Faws4_request&X-Amz-Signature=d44fd7cc02168371f03463c0f96a5794d265c4f7ae591cc9c83d569925c7c317)

## Related Issues

#1752
